src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -3361,15 +3362,7 @@ private BoundExpression BindNullCoalescingOperator(BinaryExpressionSyntax node,
                     Conversion.NoConversion, CreateErrorType(), hasErrors: true);
             }
 
-            // SPEC ERROR: The specification states:
```

Comment:
The ECMA Spec for C#5 permits the `null` literal on the left-hand side of a null coalescing operator, so this comment is now unnecessary.